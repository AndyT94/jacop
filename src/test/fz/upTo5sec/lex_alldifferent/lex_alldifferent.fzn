array [1..9] of var int: x:: output_array([1..3,1..3]) = [5,2,3,5,2,6,5,3,3];
solve  satisfy;
