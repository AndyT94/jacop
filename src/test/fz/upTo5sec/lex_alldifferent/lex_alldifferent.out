x = array2d(1..3, 1..3, [5, 2, 3, 5, 2, 6, 5, 3, 3]);
----------